Dockerfile

```diff
@@ -8,11 +8,7 @@ RUN apt update && apt install -y nginx
 
 COPY nginx.conf /etc/nginx/sites-available/default
 
-COPY package.json ./
```

Comment:
why you needed to make this change?

Reply:
`COPY . . ` will do the same. no need to do it twice

Reply:
@toliaqat created the issue